src/toil/batchSystems/parasol.py

```diff
@@ -190,7 +196,7 @@ def setEnv(self, name, value=None):
         return super(ParasolBatchSystem, self).setEnv(name, value)
 
     def __environment(self):
-        return (k + '=' + (os.environ[k] if v is None else v) for k, v in list(self.environment.items()))
+        return (k + '=' + (os.environ[k] if v is None else v) for k, v in self.environment.items())
```

Comment:
How about:
`from future.utils import listitems`
`itemlist = listitems(self.environment)`
to maintain python 3 compatibility?